azerothcore/azerothcore-wotlk
# fix(DB/Creature Loot) - Update Bloodkelp (22094) drop tables

## Changes Proposed:

This PR proposes changes to:

Updates the drop rates for the various creatures that drop Bloodkelp (22094). This was discussed a while back in #9785, but it doesn't appear any changes were implemented before the issue was closed.

## Issues Addressed:

- Addresses #9785 (which is already closed, but this isn't fixed?)

## SOURCE:

The changes have been validated through:

[WoWHead Reference Link](https://www.wowhead.com/wotlk/item=22094/bloodkelp)

## Tests Performed:

This PR has been:


## How to Test the Changes:



1. `.quest add 8970`
2. `.tel alcazisland`
3. Kill affected creatures and observe updated drop.

## Known Issues and TODO List:

None


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
